doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -70,46 +71,50 @@ Identifiers and access identifiers
   symbols, hyphens, non-breaking space, … The entry ``unicode-id-part``
   non-exhaustively includes symbols for prime letters and subscripts.
 
-  Access identifiers, written :token:`access_ident`, are identifiers prefixed by
-  `.` (dot) without blank. They are used in the syntax of qualified
-  identifiers.
+  Field identifiers, written :token:`field`, are identifiers prefixed by
+  `.` (dot) with no blank between the dot and the identifier. They are used in
+  the syntax of qualified identifiers.
 
 Numerals
-  Numerals are sequences of digits with a potential fractional part
-  and exponent. Integers are numerals without fractional nor exponent
-  part and optionally preceded by a minus sign. Underscores ``_`` can
-  be used as comments in numerals.
+  Numerals are sequences of digits with an optional fractional part
+  and exponent, optionally preceded by a minus sign. :token:`int` is an integer;
+  a numeral without fractional or exponent parts. :token:`num` is a non-negative
+  whole number.  Underscores embedded in the digits are ignored, for example:
+  ``1_000_000`` is the same as ``1000000``.
 
   .. productionlist:: coq
-     digit   : 0..9
+     numeral : `num`[. `num`][`exp`[`sign`]`num`]
+     int     : [-]`num`
      num     : `digit`…`digit`
-     integer : [-]`num`
-     dot     : .
+     digit   : 0..9
      exp     : e | E
      sign    : + | -
-     numeral : `num`[`dot` `num`][`exp`[`sign`]`num`]
 
 Strings
   Strings are delimited by ``"`` (double quote), and enclose a sequence of
   any characters different from ``"`` or the sequence ``""`` to denote the
   double quote character. In grammars, the entry for quoted strings is
-  :production:`string`.
+  :token:`string`.
+
+  .. productionlist:: coq
+     string : " `characters_opt` "
+     characters_opt : characters_opt character
```

Comment:
```suggestion
     characters_opt : `characters_opt` `character`
```

Reply:
I'm unconvinced of the usefulness of creating a `productionlist` for this given that this will still require a `:production:` in the end (for `character` instead of `string`). I feel like the textual description is enough.